tests/integration/schema/simple_test.go

```diff
@@ -150,23 +150,6 @@ func TestSchemaSimpleCreatesSchemaWithDefaultFieldsGivenEmptyType(t *testing.T)
 	testUtils.ExecuteTestCase(t, []string{"Users"}, test)
 }
 
-func TestSchemaSimpleErrorsGivenTypeWithInvalidFieldType(t *testing.T) {
```

Comment:
suggestion: Instead of removing that test, it would be nice to keep but change to the new expected error